image-loader/app/lib/imaging/FileMetadataReader.scala

```diff
@@ -91,29 +93,40 @@ object FileMetadataReader {
       }
     } getOrElse Map()
 
+  private def toLogstashFriendlyMarkers(data: Map[String, String]): LogstashMarker = {
+    val cleaned = data.collect {
+      case (key, value) => key.replaceAll("[\\[\\]:/]", "") -> value
```

Comment:
It looks like logstash can't parse `[`, `]`, `:` and `/` characters into marker keys 😢 so just remove them... its better than removing the marker entirely.

Reply:
Can we replace them with something(s)? No probs if not, would just help searching for eg. parent tags…

Reply:
just double checking as I'd like to use this pattern -- is there a reason this method is colocated with this class? I'm thinking of where to locate it in other projects, possibly in some logging utils object

Reply:
> Can we replace them with something(s)? No probs if not, would just help searching for eg. parent tags…

This is only temporary to help understand the xmp extraction issue.

>  is there a reason this method is colocated with this class?

This is only really needed in this one place at the moment and didn't want to optimise too early. Pushing it into a logging utils object would be best.

Reply:
Yeah some logging utils object will make sense for me as well, it will be easier to add tests to this utility function later if it will be needed